### PR TITLE
Fix 404 on update callout

### PR DESCRIPTION
**Pull request by basilesimon**

### CODE REVIEW

File: src/utils/api/callout.ts [hunk 1]
```diff
@@ -69,8 +69,11 @@ export async function createCallout(
 export async function updateCallout(
   slug: string,
   calloutData: UpdateCalloutData
-): Promise<any> {
-  const { data } = await axios.patch('/callout/' + slug, calloutData);
+): Promise<GetBasicCalloutData> {
+  const { data } = await axios.patch<Serial<GetBasicCalloutData>>(
```
Comment (wpf500) [inline src/utils/api/callout.ts]: Looks good but I think the `PATCH` request returns `GetMoreCalloutData`

**Comment by basilesimon** [comment]
Looks good but unfortunately doesn't fix. Really unclear what's going on: the backend appears right

https://github.com/beabee-communityrm/beabee/blob/e552a498a1eaf4364a5f2cb97662990a1d879ac9/src/api/controllers/CalloutController.ts#L157-L164

Unless it needs Authorized: admin?

**Comment by basilesimon** [comment]
🎉 This is working on `demo.beabee.io`
Ready for your review now, @wpf500

**Comment by basilesimon** [comment]
https://github.com/beabee-communityrm/beabee/blob/e552a498a1eaf4364a5f2cb97662990a1d879ac9/src/api/controllers/CalloutController.ts#L161

**Pull request merged**